### PR TITLE
Complete mobile version

### PR DESCRIPTION
**Pull request by Kwaku28**
This second part of the project is done to complete the portfolio mobile version and it includes:

1. A project part that showcases various projects
2. An about me section that informs users about skills and information about the developer
3. A contact section that includes a form for a user to reach out to the developer

Background images were added to improve the overall look of the website
Styling was done to texts to improve accessibility

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -124,3 +136,255 @@ p {
...
+  list-style: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+}
+
+.button {
```
Comment (Shisui6) [inline style.css:219]: Kindly update your button element to include button interaction styling when `hovered ` and `pressed`🛠️. You can use `.button:hover` and `.button:active` to style such interactions. Please ensure to match the design guide provided by figma 🎨

Figma design:
image

File: index.html [hunk 2]
```diff
@@ -36,24 +36,236 @@ <h2>I'm a Software Developer</h2>
...
+      <section class="about">
+        <h2>About me</h2>
+        <p>
+          I can help you build a product, feature or website. Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <button type="button" class="button">Get My Resume</button>
+
+        <div class="tools">
```
Comment (Shisui6) [inline index.html:160]: Kindly update this element to use a `<ul>` tag, as it is one of the project requirements🙏.

image

Also, ensure that each of the technology divs is also wrapped in `<li>` tags. You can do this by replacing each instance of `<div class="works">` with a `<li>` tag🫡

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Kwaku28 , 

# Requires Changes 🍀
You have done an excellent job so far! I really liked that you went ahead and finished the contact form even though you didn't need to💪

Your design is good 🖼️, and your code follows best practices. However, there are just a few issues you still need to work on.

## Highlights
- Great design ✅
- Flexbox and grid used ✅
- Good usage of semantic tags ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (wayungi) [reply style.css:219]: - [ ] As the previous reviewer suggested, please implement the button states `hover` & `pressed` 🔧

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Kwaku28 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

image


 ## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kwaku28** [comment]
Could you please recheck, the review was done on the wrong branch🙂

File: style.css (hunk 1 above)
Comment (Kwaku28) [reply style.css:219]: Could you please recheck, the review was done on the wrong branch🙂

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @Kwaku28     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**